rpm/SPECS/perl-Test-Simple.spec

```diff
@@ -1,5 +1,5 @@
 Name:           perl-Test-Simple
-Version:        1.302175
+Version:        1.302190
 Release:        3%{?dist}
```

Comment:
```suggestion
Release:        1%{?dist}
```

Reply:
also need to change the deb script

Reply:
@zhuizhuhaomeng  I Know a few about deb. current I can't find any  script about  Test-Simple